galaxy_ng/app/api/v3/viewsets/collection.py

```diff
@@ -180,3 +186,64 @@ def get(self, request, *args, **kwargs):
         metrics.collection_artifact_download_failures.labels(status=response.status_code).inc()
         raise APIException('Unexpected response from content app. '
                            f'Code: {response.status_code}.')
+
+
+class CollectionVersionMoveViewSet(ViewSet):
+    permission_classes = GALAXY_PERMISSION_CLASSES + [
+        permissions.IsNamespaceOwner
+    ]
+
+    def move_content(self, request, *args, **kwargs):
+        """Remove content from source repo and add to destination repo.
+
+        Creates new RepositoryVersion of source repo without content included.
+        Creates new RepositoryVersion of destination repo with content included.
+        """
+
+        try:
+            collection_version = CollectionVersion.objects.get(
+                namespace=self.kwargs['namespace'],
+                name=self.kwargs['name'],
+                version=self.kwargs['version'],
+            )
+        except ObjectDoesNotExist:
+            raise NotFound('Collection version not found')
```

Comment:
Do user facing errors or log messages include info about which collection version was not found?

If not, that info should be added to the messages.

Reply:
Ah, just saw this comment. The message now has the collection version added to it.